src/components/AnchorForCommentsOnly/index.native.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {Linking} from 'react-native';
+import BaseAnchorForCommentsOnly from './BaseAnchorForCommentsOnly';
+import type AnchorForCommentsOnlyProps from './types';
+
+function AnchorForCommentsOnly({onPress, href, ...props}: AnchorForCommentsOnlyProps) {
+    const onLinkPress = () => (typeof onPress === 'function' ? onPress() : Linking.openURL(href ?? ''));
```

Comment:
Let's use a `if/else` block, instead of the ternary operator for better readability.